### DISCUSSION TITLE
Uninstall from Appcenter does not work

### DISCUSSION THREAD

**Discussion by sysfu** [elementary/mail#898] [Q&A]
I want to uninstall mail but it apparently cannot be done from Appcenter. Clicking the uninstall button appears to work, then when it completes, the app remains installed.

uninstalling the io.elementary.mail pkg in the terminal using the 'apt remove' command works as expected.

**Comment by 1280px** [comment]
unrel but this is a common problem with appcenter, sometimes it even struggles to uninstall user-installed flatpaks
